Remove gucs specific to planning

gayyappan opened the pull request:
These gucs are for internal use only.

ryanbooz commented:
It's up to you how you want to handle this PR, but I had already moved this file and removed some of these based on @gayyappan 's comment in Slack. Might be easier just to review my chances to make sure I remove all of them?

ryanbooz reviewed (changes requested):
I had already moved this page in another branch to fix some other 2.0 documentation changes. Initial PR #565

gayyappan commented:
> It's up to you how you want to handle this PR, but I had already moved this file and removed some of these based on @gayyappan 's comment in Slack. Might be easier just to review my chances to make sure I remove all of them?

That works. I'll close this PR.
